### PR TITLE
fix /statistic issue by modifying queries in statistic.ts

### PR DESCRIPTION
**Pull request by taylorziegler**
Attempt to fix /statistic issue by resolving a bug in Supervisor.ts where the finishedAt field is not set for any jobs.

### CODE REVIEW

**Comment by taylorziegler** [comment]
The issue was that we were using TIMESTAMPDIFF on a non-timestamp value in statistic.ts. I modified it to just be the difference in seconds.

File: src/Supervisor.ts [hunk 1]
```diff
@@ -129,8 +130,14 @@ class Supervisor {
 
       // connect ssh & run
       try {
-        if (!ssh.connection.isConnected())
-          await ssh.connection.connect(ssh.config);
+        var ct = 1;
```
Comment (alexandermichels) [inline src/Supervisor.ts]: Can we remove this exponential backoff stuff from the PR and put it in another branch/PR?

**Review by alexandermichels — APPROVED** [review]
Looks fine, but please try to remove remove the exponential backoff code and put it in another branch/PR.

File: src/Supervisor.ts [hunk 2]
```diff
@@ -72,6 +72,7 @@ class Supervisor {
               .execute();
             continue;
           }
+
```
Comment (mitkotak) [inline src/Supervisor.ts:75]: redundant space

File: src/Statistic.ts [hunk 3]
```diff
@@ -59,4 +56,4 @@ export default class Statistic {
       return null;
     }
   }
-}
+}
```
Comment (mitkotak) [inline src/Statistic.ts:59]: can you bring back the space

File: src/Statistic.ts [hunk 4]
```diff
@@ -10,27 +10,24 @@ export default class Statistic {
...
         { id: jobId }
       )
       .getRawOne();
-    if (statistic) {
-      return parseInt(statistic["STATISTIC"]);
-    } else {
-      return null;
-    }
-  }
+
```
Comment (mitkotak) [inline src/Statistic.ts:20]: Is it helpful to default to null or something else in case there's any error ?

**Review by mitkotak — CHANGES_REQUESTED** [review]
Good job ! Some minor nitpicks

**Review by alexandermichels — APPROVED** [review]
LGTM

**Review by mitkotak — APPROVED** [review]
LGTM !

**Pull request merged**